### PR TITLE
[LoongArch][Clang] Add tests for #110834

### PR DESCRIPTION
**Pull request by xry111**

### CODE REVIEW

File: clang/test/Headers/lasxintrin.c [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// REQUIRES: loongarch-registered-target
```
Comment (SixWeining) [inline clang/test/Headers/lasxintrin.c:1]: Seems this is not required.

File: clang/test/Headers/lasxintrin.c [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+// RUN: %clang_cc1 %s -fsyntax-only -triple loongarch64-linux -target-feature +lasx
```
Comment (SixWeining) [inline clang/test/Headers/lasxintrin.c:2]: Should we emphasize linux?

Comment (SixWeining) [inline clang/test/Headers/lasxintrin.c:2]: Add `not` and leave a `FIXME` or `TODO`. Then remove it in next commit.

File: clang/test/Headers/lasxintrin.c (hunk 1 above)
Comment (xry111) [reply clang/test/Headers/lasxintrin.c]: Dropped.

File: clang/test/Headers/lasxintrin.c (hunk 2 above)
Comment (xry111) [reply clang/test/Headers/lasxintrin.c]: -linux dropped.  I just mistakenly thought a triple had to have the os field.

**Review by SixWeining — APPROVED** [review]

**Pull request merged**